documentation/design-docs/dotnet-tools.md

```diff
@@ -404,38 +406,25 @@ OPTIONS
 
 COMMANDS
 
-    collect   Capture one or more dumps (core files on Mac/Linux) from a process
+    collect   Capture dumps from a process
     analyze   Starts an interactive shell with debugging commands to explore a dump
 
 COLLECT
 
     dotnet-dump collect -p|--process-id <pid>
-                        [-h|--help]
-                        [--interval-sec <seconds>]
-                        [--number <number_of_dumps>]
-                        [-o|--output <output_dump_path>]
-                        [--type <dump_type>]
+                       [-h|--help]
+                       [-o|--output <output_dump_path>]
+                       [--type <dump_type>]
 
     Capture one or more dumps (core files on Mac/Linux) from a process
 
     -p, --process-id
-        The process to collect dumps from
-
-    -h, --help
```

Comment:
I don't think we should remove 'help' option

Reply:
System.CommandLine doesn't put the -h, --help option in the help.

Reply:
But I can open an issue to change this.

Reply:
Ah ok makes sense. If thats the default behavior I'm happy : )

Reply:
It's the current default behavior but that doesn't mean it's correct or unsurprising. We can certainly change it. 

The current behavior is a bit of a side effect of the fact that `-h` isn't implemented as an option. This is because, logically, `-h` should be an option of *every* command in the hierarchy, and unlike most options, it acts like a command or verb. For this reason we implemented handling of `-h` (and a bunch of other tokens that commonly mean help in Windows and POSIX command lines) using middleware.

Reply:
I'm going to leave --help in the spec fix this later (either by changing the spec or filing an issue on System.CommandLine).

Reply:
@jonsequitur the default help builder doesn't format long option descriptions very well. I think it is because the maxWidth parameter to the HelpBuilder isn't being set and there is no way to set it for the default help builder.  I guess I could create my own help builder instance with the options I want but then I would have to hook it in as middleware myself (I think).

Also (this is for the interactive command "help"), it would be nice if the IHelpBuilder instance is available through a public property in the InvocationContext or somewhere. I know (I use it) it is available through the MethodBinder as a parameter to an invoke function, but it would be easier to be in one of the "contexts".

I'm currently using version 0.1.0-alpha-63807-01.